deployment/terraform/dns.tf

```diff
@@ -73,3 +73,20 @@ resource "aws_route53_record" "www_ipv6" {
     evaluate_target_health = false
   }
 }
+
+resource "aws_route53_record" "ses_verification" {
+  zone_id = "${aws_route53_zone.external.zone_id}"
+  name    = "_amazonses.${var.r53_public_hosted_zone}"
+  type    = "TXT"
+  ttl     = "300"
+  records = ["${aws_ses_domain_identity.app.verification_token}"]
+}
+
+resource "aws_route53_record" "ses_dkim" {
+  count   = 3
```

Comment:
Not super important, but it would be interesting to try and get the length of `dkim_tokens` with `length()` vs. hardcoding 3 here.

Reply:
I tried this and received this error when running `$ terraform plan`:

```bash
Error: Error refreshing state: 1 error(s) occurred:

* aws_route53_record.ses_dkim: aws_route53_record.ses_dkim: value of 'count' cannot be computed
```

When Terraform says "computed" here it means a value that isn't known until its associated resource is created. It works on a subsequent run because the resource has already been created by then and so the value is known.

All the documentation I've found online seems to suggest that there will only ever be 3 DKIM tokens generated, and the example in the Terraform docs hardcodes 3 as well. 

See: 
* https://github.com/awsdocs/amazon-ses-developer-guide/blob/master/doc-source/easy-dkim-dns-records.md
* https://www.terraform.io/docs/providers/aws/r/ses_domain_dkim.html#example-usage

Wondering what your thoughts are.